incubator/external-dns/values.yaml

```diff
@@ -0,0 +1,68 @@
+
+# Details about the image to be pulled.
+image:
+  repository: registry.opensource.zalan.do
+  name: teapot/external-dns
+  tag: v0.3.0
+  pullPolicy: IfNotPresent
+
+# This controls which types of resource external-dns should 'watch' for new
+# DNS entries.
+source:
+  fake: false
+  ingress: true
+  service: true
```

Comment:
This could maybe be done via
```yaml
  sources:
  - service
  - ingress
```
and then just iterate over it writing out `--source=${i}` lines in the args section of the deployment template?